stories/button/container.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { Button, Typography } from 'src/shared-components';
+
+// eslint-disable-next-line react/prop-types
+export default ({
```

Comment:
Let's avoid `export default`--it hurts discoverability and also results in anonymous components in the React devtools. 

```
const Container = ({}).

export default Container;
```

(It also looks like we can remove the eslint disable).